### ISSUE TITLE
Is this project abandoned?

### ISSUE THREAD

**Issue by In-line** [yrnkrn/zapcc#35]
It didn't receive any updates since 2018

**Comment by yrnkrn** [comment]
zapcc is no longer commercially developed, so it's on a free time basis...

**Comment by atomsymbol** [comment]
- Is this (yrnkrn/zapcc) the official repository for pull requests?
- It would be more convenient if the main URL was https://github.com/zapcc/zapcc (https://github.com/zapcc is currently free for creating a zapcc github organization)

**Comment by yrnkrn** [comment]
I created a zapcc org at https://github.com/zapcc. Thanks!
How would having the repository at https://github.com/zapcc/zapcc more convenient?
(moving would break existing links to here)

**Comment by tux3** [comment]
>moving would break existing links to here

I don't have an opinion on renaming, but note that if you move the repo, Github will [keep the old links working](https://github.blog/2013-05-16-repository-redirects-are-here/) and automatically redirect to the new url.

**Comment by merbst** [comment]
Personally, my knowledge of Microsoft makes me unwilling to trust the reliability of GitHub under its control, so I was pleased to see how easy GitLab.com has made transferring the canonical master location of your GitHub projects, and they even provide continuing broadcasting to synchronize all future changes to the preexisting GitHub projects.

**Comment by JeffCarpenter** [comment]
I would *love* to see this project funded by a Kickstart or Patreon type arrangement. And it seems like a fairly easy sell. I mean the potential cost savings worldwide are absolutely enormous.

**Comment by yrnkrn** [comment]
Yep, saving highly-paid C++ developers time made **lots** of sense to us at Ceemple software where we invested much effort and money to get zapcc up and working. It **should** be worth $1000/year to a company to save half the compilation time, right?
In realty while zapcc generated lots of interest from everyone, no one was actually buying it or willing to invest in keeping zapcc development alive. At some point we ran out of funds for the couple of C++ software developers employed.
Offers are welcome!
